### PR TITLE
Add release and EOL dates to Drush versions table

### PR DESCRIPTION
**Pull request by greg-1-anderson**
Related to #4314.

### CODE REVIEW

**Comment by greg-1-anderson** [comment]
Looks like the Drush 9 tests are failing due to `core_version_requirement` becoming a required element. Not related to this PR.

I don't know if all of the extra fields in this table are necessary, but it's interesting. Folks want to know when Drush versions are going to be EOL, though, so we should try to convey that information at least for the active versions.

**Comment by goba** [comment]
Thanks, this would be a really useful resource to point people to.

**Comment by weitzman** [comment]
Lets remove  the Release column in the name of brevity/simplicity.

File: docs/install.md [hunk 1]
```diff
@@ -80,4 +93,7 @@ Drupal Compatibility
     <tr>
         <td>✓</td> <td>Compatible but no longer supported</td>
     </tr>
+    
 </table>
+
+❶: EOL date for Drush 8 tbd, but estimated to be in concert with <a href="https://www.drupal.org/psa-2019-02-25">Drupal 7 EOL</a>.</td>
```
Comment (heddn) [inline docs/install.md]: maybe spell out TBD for non english as first language folks? At least capitalize for clarity its an acronym.
```suggestion
❶: EOL date for Drush 8 TBD, but estimated to be in concert with <a href="https://www.drupal.org/psa-2019-02-25">Drupal 7 EOL</a>.</td>
```

File: docs/install.md [hunk 2]
```diff
@@ -33,36 +34,48 @@ Drupal Compatibility
     <td> Drush 10 </td>
     <td> master </td>
     <td> 7.1+ </td>
+    <!-- Released Oct 2019 -->
+    <td> <i>tbd</i> </td>
```
Comment (heddn) [inline docs/install.md]:
```suggestion
    <td> <i>TBD</i> </td>
```

Comment (greg-1-anderson) [reply docs/install.md]: I think it is contextually clear that the EOL date for the most recent version is not set yet. Uppercasing TBD is fine though.

**Pull request merged**